test/e2e/user_workload_monitoring_test.go

```diff
@@ -322,8 +265,9 @@ func assertMetricsForMonitoringComponents(t *testing.T) {
 		"prometheus-user-workload-thanos-sidecar": 2,
 	} {
 		t.Run(service, func(t *testing.T) {
+			t.Parallel()
 			f.ThanosQuerierClient.WaitForQueryReturn(
-				t, time.Minute, fmt.Sprintf(`count(up{service="%s",namespace="openshift-user-workload-monitoring"} == 1)`, service),
+				t, 5*time.Minute, fmt.Sprintf(`count(up{service="%s",namespace="openshift-user-workload-monitoring"} == 1)`, service),
```

Comment:
This runs in parallel now, some checks that used to run at the end may need more time now.